airflow/bin/cli.py

```diff
@@ -283,6 +292,43 @@ def _tabulate(pools):
         log.info(_tabulate(pools=pools))
 
 
+def pool_import_helper(filepath):
+    with open(filepath, 'r') as poolfile:
+        pl = poolfile.read()
+    try:
+        d = json.loads(pl)
+    except Exception:
+        print("Invalid pool file.")
+    else:
+        try:
+            pools = []
+            n = 0
```

Comment:
We don't need this `n` counter, we can also do `len(pools)`

Reply:
I reason I put `n` counter here is I am following the same pattern here as Variable to give the user same experience https://github.com/apache/incubator-airflow/blob/0ba510e238ebb197bce9afccdb06b88871c4a7e6/airflow/bin/cli.py#L375-L382

So it would be something like `"1 of 2 pool(s) successfully updated."`